ns1/rest/zones.py

```diff
@@ -110,6 +110,71 @@ def search(
             errback=errback,
         )
 
+    def list_versions(
+            self,
+            zone,
+            callback=None,
+            errback=None
+    ):
+        request = "{}/{}/versions".format(self.ROOT, zone)
```

Comment:
This follows the pattern we use elsewhere, but Python f-strings are now in every supported version. (They were introduced in Python 3.6, but Python 3.7 and earlier are all deprecated.)

Using f-strings for this would look like this:

```suggestion
        request = f"{self.ROOT}/{zone}/versions"
```

I find it a lot easier to read, but I can understand that maybe we do not want to mix styles.